src/extensions/mvs/load-extensions/is-hidden-custom-state.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2025 mol* contributors, licensed under MIT, See LICENSE file for more info.
+ *
+ * @author David Sehnal <david.sehnal@gmail.com>
+ */
+
+import { MolstarLoadingExtension } from '../load';
+
+
+export const IsHiddenCustomStateExtension: MolstarLoadingExtension<{}> = {
+    id: 'mvs/is-hidden-custom-state',
```

Comment:
I conceived the prefix of the id as a "scope", something like name of the organization that maintains the extension or name of the project it is used for. `mvs/` makes little sense here, as all extensions will be for MVS. 

Also, perhaps `-custom-state` is superfluous as all these extensions are kinda expected to work with custom state.

So maybe `wwpdb/is-hidden` (or `wwpdb/hide-nodes' or similar)?